filebeat/harvester/log.go

```diff
@@ -99,6 +99,13 @@ func (h *Harvester) Harvest() {
 			event.Text = &text
 			event.JSONFields = message.Fields
 			event.EventMetadata = h.config.EventMetadata
+			if message.Fields != nil {
+				if event.EventMetadata.Fields != nil {
+					event.EventMetadata.Fields.Update(message.Fields)
```

Comment:
is this overwriting EventMetadata.Fields or does it create a new `common.MapStr` internally? EventMetadata directly uses config.EventMetadata.